src/main/java/microsoft/exchange/webservices/data/core/EwsSSLProtocolSocketFactory.java

```diff
@@ -23,24 +23,26 @@
 
 package microsoft.exchange.webservices.data.core;
 
+import org.apache.http.conn.ssl.DefaultHostnameVerifier;
```

Comment:
@vbauer unfortunately I missed updating `<httpclient.version>` to `4.4.1` because it is for some reasons not listed at [mvn repo](http://mvnrepository.com/artifact/org.apache.httpcomponents/httpclient). Maybe you can add this since it is thematically related (would save me creating another PR for this :trollface: )?